python/TestHarness/suppressions/errors.supp

```diff
@@ -112,3 +112,10 @@
    fun:mca_base_component_find
    fun:mca_base_framework_components_register
 }
+{
+   Super_LU
+   Memcheck:Cond
+   ...
+   fun:MatLUFactorNumeric_SuperLU_DIST
```

Comment:
Oh, I thought this was a demangled name, then I realized it came from a C compiler.